.github/workflows/main.yml

```diff
@@ -37,29 +33,42 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - uses: actions/setup-node@v1
+            - uses: actions/checkout@v2
+            - name: Fetch and Diff
+              id: diff
+              run: |
+                  git fetch --no-tags --depth=1 origin ${{ github.event.pull_request.base.sha }}
+                  echo "##[set-output name=packages;]$(git diff --quiet ${{ github.event.pull_request.base.sha }} '**/packages*')"
```

Comment:
Clever trick! Didn't know about this syntax.

Reply:
Took me way too long to figure out. Some of the hokey work arounds for gh actions are so odd